src/dev/impl/DevToys/Views/Tools/Converters/Timestamp/TimestampToolPage.xaml

```diff
@@ -9,41 +9,206 @@
     mc:Ignorable="d"
     d:DesignHeight="400"
     d:DesignWidth="600">
-    <StackPanel Margin="0,0,0,16" Spacing="12">
+
+    <StackPanel
+        Margin="0,0,0,16"
+        Spacing="12">
+
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <AdaptiveTrigger
+                            MinWindowWidth="640" />
+                    </VisualState.StateTriggers>
+
+                    <VisualState.Setters>
+                        <Setter
+                            Target="CurrentDateGridBlock.Grid.ColumnSpan"
+                            Value="1" />
+
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.Row"
+                            Value="0" />
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.Column"
+                            Value="1" />
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.ColumnSpan"
+                            Value="1" />
+                    </VisualState.Setters>
+                </VisualState>
+            </VisualStateGroup>
+        </VisualStateManager.VisualStateGroups>
+
+
         <StackPanel.ChildrenTransitions>
             <TransitionCollection>
-                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+                <EntranceThemeTransition
+                    IsStaggeringEnabled="True"
+                    FromVerticalOffset="50" />
             </TransitionCollection>
         </StackPanel.ChildrenTransitions>
 
-        <StackPanel Margin="0,12,0,0">
-
+        <StackPanel
+            Margin="0,12,0,0">
             <muxc:InfoBar
                 Severity="Warning"
                 Message="{x:Bind ViewModel.Strings.InvalidValue}"
                 IsOpen="{x:Bind ViewModel.IsInputInvalid, Mode=OneWay}"
-                IsClosable="False"/>
+                IsClosable="False" />
+
+            <TextBlock
```

Comment:
I have trouble understanding the goal of this `Current date and time` label. It's just above `Timezone` and it doesn't show constantly the current time in the UI.